### ISSUE TITLE
Cannot remove divider above Privacy and Terms links in core-2

### ISSUE THREAD

**Issue by danilofuchs** [clerk/javascript#3275]
### Preliminary Checks

- [X] I have reviewed the documentation: https://clerk.com/docs

- [X] I have searched for existing issues: https://github.com/clerk/javascript/issues

- [X] I have not already reached out to Clerk support via email or Discord (if you have, no need to open an issue here)

- [X] This issue is not a question, general help request, or anything other than a bug report directly related to Clerk. Please ask questions in our Discord community: https://clerk.com/discord.

### Reproduction

https://codesandbox.io/p/devbox/clerk-divider-vttkym?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clvgs6otf00062v6ktxlq8jn4%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clvgs6otf00022v6ks55vium1%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clvgs6otf00042v6ke0jdu39l%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clvgs6otf00052v6kpjics2ia%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clvgs6otf00022v6ks55vium1%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clvgs6otf00012v6k3s6oan33%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252FREADME.md%2522%257D%255D%252C%2522id%2522%253A%2522clvgs6otf00022v6ks55vium1%2522%252C%2522activeTabId%2522%253A%2522clvgs6otf00012v6k3s6oan33%2522%257D%252C%2522clvgs6otf00052v6kpjics2ia%2522%253A%257B%2522id%2522%253A%2522clvgs6otf00052v6kpjics2ia%2522%252C%2522activeTabId%2522%253A%2522clvgs73ey002i2v6kt2x991nt%2522%252C%2522tabs%2522%253A%255B%257B%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A3000%252C%2522id%2522%253A%2522clvgs73ey002i2v6kt2x991nt%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522path%2522%253A%2522%252F%2522%257D%255D%257D%252C%2522clvgs6otf00042v6ke0jdu39l%2522%253A%257B%2522id%2522%253A%2522clvgs6otf00042v6ke0jdu39l%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clvgs6otf00032v6kcis2046w%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522dev%2522%257D%255D%252C%2522activeTabId%2522%253A%2522clvgs6otf00032v6kcis2046w%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D

### Publishable key

pk_test_Y3JlZGlibGUtcmFiYml0LTUuY2xlcmsuYWNjb3VudHMuZGV2JA

### Description

I want to remove this divider above Privacy and Terms links at the `<SignIn />` component, but there is no exposed class for it.

![image](https://github.com/clerk/javascript/assets/17657014/11a9bb5f-69ed-4dfe-810e-a753b3cc4349)
![image](https://github.com/clerk/javascript/assets/17657014/58481575-774d-47a5-bda1-64d98bc7b777)

### Environment

```Shell
`@clerk/nextjs 5.0.2`
```

**Comment by panteliselef** [comment]
Hey @danilofuchs could you give this a try ?
```tsx
<ClerkProvider
  appearance={{
    elements: {
      footer: {
        '>:nth-of-type(2)': {
          borderTopWidth: '0px',
        },
      },
    },
  }}
>
```

**Comment by danilofuchs** [comment]
This works @panteliselef!
Not sure if this is the best API though

**Comment by panteliselef** [comment]
That is correct, having said that tho, due to css specificity simply introducing a class will not fix the issue, we would have to remove the previous css rule which we are considering a breaking change.

**Comment by panteliselef** [comment]
I've added an internal ticket for this for our next major, thanks for reporting

**Comment by clerk-cookie** [comment]
This issue has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.
